conf/airframes/MC/mc_ardrone2_opticflow_swarm0.xml

```diff
@@ -0,0 +1,281 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="ardrone2_opticflow">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="ardrone2">
+      <!-- HOST is the Name or IP of your AC -->
+      <configure name="HOST" value="192.168.40.200"/>
+      <!-- MODEM_HOST is where the data should go to, for short normally your laptop IP address -->
+      <configure name="MODEM_HOST" value="192.168.40.146"/>
+      <configure name="MODEM_BROADCAST" value="FALSE"/>
+    </target>
+
+    <!--target name="nps" board="pc">
+      <module name="fdm" type="jsbsim"/>
+    </target-->
+
+    <define name="USE_SONAR" value="TRUE"/>
+    <!-- <define name="USE_BARO_BOARD" value="FALSE"/> -->
+
+  <!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="ardrone2"/>
+    <module name="imu" type="ardrone2"/>
+    <module name="gps" type="datalink"/>
+    <module name="stabilization" type="indi_simple"/>
+    <module name="ahrs" type="int_cmpl_quat">
+      <configure name="USE_MAGNETOMETER" value="TRUE"/>
+      <configure name="AHRS_USE_GPS" value="FALSE"/>
+      <!--configure name="AHRS_USE_GPS_HEADING" value="TRUE"/-->
+    </module>
+    <module name="ins" type="extended"/>
+  </firmware>
+
+ <section name="RELATIVELOCALIZATIONFILTER" prefix="">
+  <define name="RSSINOISE" value="5.0"/>  <!-- (Expected) Noise on RSSI -->
+  <define name="SPEEDNOISE" value="0.2"/> <!-- (Expected) Noise on other measurements -->
+  <define name="NUAVS" value="5"/>        <!-- Maximum number of UAVs to track -->
+  <define name="BLUEGIGA_DONGLE" value="TRUE" />
+ </section>
+
+  <modules main_freq="512">
+    <module name="bat_voltage_ardrone2"/>
```

Comment:
We are phasing out separate firmware and modules sections, can you move these to the firmware section?